staging/com.smartnews/dataset.tt

```diff
@@ -0,0 +1,90 @@
+dataset @com.smartnews language "en" {
+    query := @com.smartnews.top_articles()
+    #_[utterances=["smartnews",
+                   "smart news",
+                   "smartnews top",
+                   "smart news top",
+                   "news",
+                   "top news",
+                   "top smartnews",
+                   "top smart news",
+                   "news from smartnews",
+                   "news on smartnews",
+                   "breaking news",
+                   "smart news",
+                   "today news",
+                   "popular news",
+                   "important news",
+                   "today's news",
+                   "global news",
+                   "hot topic",
+                   "hot news",
+                   "high rank news"]]
+    #[id=61614268]
+    #[name="TopArticles"];
+
+    query (p_counter :Number) := @com.smartnews.top_articles(counter=p_counter)
+    #_[utterances=["$p_counter smartnews",
+                   "$p_counter smart news",
+                   "$p_counter smartnews top",
+                   "$p_counter smart news top",
+                   "$p_counter news",
+                   "$p_counter top news",
+                   "$p_counter top smartnews",
+                   "$p_counter top smart news",
+                   "$p_counter news from smartnews",
+                   "$p_counter news on smartnews",
+                   "$p_counter breaking news",
+                   "$p_counter smart news",
+                   "$p_counter today news",
+                   "$p_counter popular news",
+                   "$p_counter important news",
+                   "$p_counter today's news",
+                   "$p_counter global news",
+                   "$p_counter hot topic",
+                   "$p_counter hot news"]]
```

Comment:
I'm not sure of all these phrases sound very natural...

Reply:
I agree. So the general guideline is to just list out the most common/nature ways? Do we need to consider abnormal cases?